helm/blueapi/templates/configmap.yaml

```diff
@@ -5,7 +5,10 @@ metadata:
 data:
   config.yaml: |-
     {{- toYaml .Values.worker | nindent 4 }}
```

Comment:
Hmm, what would happen if my `values.yaml` looked something like this?

```yaml
initContainer:
    enabled: True
    scratch:
        root: /foo

worker:
    scratch:
        root: /bar
```

I think this would produce two `scratch`s, which would turn this into invalid yaml. Could you write a test to see what happens? Ideally the chart should fail to render, if that's hard then one `scratch` should win out over the other in a predictable way.

Reply:
Yeah, that could work